devicetypes/smartthings/zwave-metering-dimmer.src/zwave-metering-dimmer.groovy

```diff
@@ -270,6 +310,62 @@ def normalizeLevel(level) {
 	level == 99 ? 100 : level
 }
 
+def getAeotecNanoDimmerDefaults() {
```

Comment:
```
def getAeotecNanoDimmerDefaults() {[
	1:0,
	2:99
]}
```
or
```
def getAeotecNanoDimmerDefaults() {
	[
		1:0,
		2:99
	]
}
```
would be more elegant

Reply:
also I'd change name of this getter to actually describe which defaults it stands for (or at least add a proper comment)

Reply:
ok, fixed